core/serialization/blocks.js

```diff
@@ -52,9 +52,6 @@ exports.ConnectionState = ConnectionState;
  *     y: (number|undefined),
```

Comment:
Not from this PR, but can we remove the TODO and the `/* eslint-disable no-use-before-define */` above?

Reply:
We can, but lint still isn't fixed, so we'd have to reorganize the file :/ Right now higher level functions are defined before lower level procedures, and we'd have to flip it.

Reply:
Ahhh, makes sense. Do you plan on fixing before merging in to master? If not, then we should just make an issue to update it so we have something to reference in the TODO.

Reply:
I can link it to https://github.com/google/blockly/issues/5160

Reply:
Yup, sounds great : )